dispatcher/backend/src/routes/requested_tasks/requested_task.py

```diff
@@ -290,9 +295,23 @@ def get(
         requested_task_id: UUID,
         token: AccessToken.Payload = None,
     ):
-        requested_task = dbm.RequestedTask.get(session, requested_task_id, TaskNotFound)
+        requested_task = dbm.RequestedTask.get(
+            session, requested_task_id, RequestedTaskNotFound
+        )
         resp = RequestedTaskFullSchema().dump(requested_task)
 
+        # also fetch all requested tasks IDs to compute estimated task rank ; this is
+        # only an estimation since logic to start tasks is more complex than simple
+        # sorting but it gives a good indicator of when the requested task might
+        # be expected to start (soon or in a long time) especially for zimit.kiwix.org
```

Comment:
This is misleading. It can only be a fair estimation on zimit ; not _especially for zimit_.
What makes it usable there is:
- no existing `duration` because schedule is created on-demand
- identical resource provisioning for all tasks
I think source-code comment should be precise about this